doc/releases/migration-guide-3.5.rst

```diff
@@ -170,6 +170,34 @@ Required changes
     };
 
 
+* GPIO drivers not implementing IRQ related operations must now provide
+  ``NULL`` to the relevant operations: ``pin_interrupt_configure``,
+  ``manage_callback``, ``get_pending_int``. The public API will return
```

Comment:
nit/question: was there a reason not to link to the c:func's, or just an oversight?

Reply:
sry missed the question, maybe do a followup if it has to be fixed

Reply:
they're not c functions, but driver ops